### ISSUE TITLE
What does the hash decay loss do? 

### ISSUE THREAD

**Issue by Saoyu99** [SuLvXiangXin/zipnerf-pytorch#12]

**Comment by SuLvXiangXin** [comment]
@Saoyu99 hash decay loss implement the weight decay loss in Zipnerf Section4 Paragraph3.

**Comment by Saoyu99** [comment]
Thanks for your great work！！
